### ISSUE TITLE
Link back to main documentation page

### ISSUE THREAD

**Issue by Jeeppler** [Daimler/sechub#368]
Once a user is on any documentation page, the user can only go back to the main documentation page by using the back button in the browser or by going to `https://daimler.github.io/sechub`. Any reader of the documentation should be able to quickly go back to the main documentation page. A simple link would be sufficient.

**Comment by sven-dmlr** [comment]
The link is already present on top of every SecHub documentation document:
`You can get more information about from {webPage}[{sechub} web page]  .`
(sechub-doc/src/docs/asciidoc/documents/shared/about_sechub.adoc)

@Jeeppler Are you fine with the wording - or do you have a better suggestion?

**Comment by Jeeppler** [comment]
Yeah, you are right, there is indeed a link as part of the text. However, that is terrible from a user point of view. I would expect that I can click on the logo or really have a link/button back to the main page.

This is how it looks in GitHub when I click on a notification:

![image](https://user-images.githubusercontent.com/2007739/200005944-e2ac2e58-c1d0-4a27-ac72-17dee0a9fa30.png)

**Comment by sven-dmlr** [comment]
Made logo clickable. See PR https://github.com/mercedes-benz/sechub/pull/1737
